src/commands/deployImage/imageSource/containerRegistry/acr/createAcr/createAcr.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { LocationListStep, ResourceGroupCreateStep } from "@microsoft/vscode-azext-azureutils";
+import { AzureWizard, AzureWizardExecuteStep, AzureWizardPromptStep, IActionContext, createSubscriptionContext, nonNullValue, subscriptionExperience } from "@microsoft/vscode-azext-utils";
+import { AzureSubscription } from "@microsoft/vscode-azureresources-api";
+import { ext } from "../../../../../../extensionVariables";
+import { createActivityContext } from "../../../../../../utils/activityUtils";
+import { localize } from "../../../../../../utils/localize";
+import { ICreateAcrContext } from "./ICreateAcrContext";
+import { RegistryCreateStep } from "./RegistryCreateStep";
+import { RegistryNameStep } from "./RegistryNameStep";
+import { SkuListStep } from "./SkuListStep";
+
+export async function createAcr(context: IActionContext, node?: { subscription: AzureSubscription }): Promise<void> {
+    const subscription = node?.subscription ?? await subscriptionExperience(context, ext.rgApiV2.resources.azureResourceTreeDataProvider);
+
+    const wizardContext: ICreateAcrContext = {
+        ...context,
+        ...createSubscriptionContext(subscription),
+        ...(await createActivityContext())
+    };
+
+    const title: string = localize('createAcr', "Create container registry");
+
+    const promptSteps: AzureWizardPromptStep<ICreateAcrContext>[] = [
+        new RegistryNameStep(),
+        new SkuListStep(),
+    ];
+
+    const executeSteps: AzureWizardExecuteStep<ICreateAcrContext>[] = [
+        new ResourceGroupCreateStep(),
```

Comment:
Should we use a `ResourceGroupListStep` instead of the create step here?  I think it might be nice to also allow the user to choose their own resource group rather than force a new one to be created each time 🤔

Reply:
Yeah, I've been meaning to ask Nathan about that actually.  My guess is that it was determined to just always skip prompting since it is the top level container apps resource, and likely the other resources would also sit in that same resource group?

For this one I took a sneak peak at how Docker implemented it, and they are using a list step too I believe, so might be worth trying to keep it similar 🤔

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/f352a475-da27-40e0-b849-cb9b559dff42)

Reply:
Depends on if we think users will care about what RG they create the registry in. Are there benefits from putting it in the same RG as other resources?

Reply:
As far as I know it doesn't make a difference which RG the registry is in. You can still use images from registry's outside of the resource group the CA is in but not sure about other use cases.

Reply:
I'm happy with however way you end up implementing, just thought to bring it up for discussion